students/hyska_monika/lesson_01_basics/ClockFace1.py

```diff
@@ -0,0 +1,11 @@
+# The program print the angle (in degrees) of the hour hand on the clock
+
+H = int(input("Put hours: "))
```

Comment:
english: rather "Enter" or just "Hours:", put is mostly used in phrasal verbs, it is rarely used alone without preposition.